src/client/App.css

```diff
@@ -1,3 +1,193 @@
 * {
     box-sizing: border-box;
-}
\ No newline at end of file
+}
+
+.App {
```

Comment:
Same comment about consistency as on this PR: https://github.com/rebeccahockley/hyf-homework/pull/21#discussion_r891086980

Try to be consistent, and use the same naming convention everywhere. Either all classes start with uppercase or lowercase (in the react community, I think having everything start with lowercase is preferred)